status/reporters.go

```diff
@@ -452,6 +452,14 @@ func (r *GitHubReporter) ReportStatusForSnapshot(k8sClient client.Client, ctx co
 		return fmt.Errorf("sha label not found %q", gitops.PipelineAsCodeSHALabel)
 	}
 	integrationTestStatusDetails := statuses.GetStatuses()
+	latestUpdateTime, _ := gitops.GetLatestUpdateTime(snapshot)
+	newLatestUpdateTime := latestUpdateTime
+	if err != nil {
+		logger.Error(err, "failed to get latest update annotation for snapshot",
+			"snapshot.NameSpace", snapshot.Namespace, "snapshot.Name", snapshot.Name)
+		return err
```

Comment:
Maybe instead returning error here, we could just use 0 timestamp and let it selfheal

Reply:
Do you agree we keep the error msg? or we want to log it as info?

Reply:
error sounds fine